json_logger/formatter.py

```diff
@@ -10,23 +10,29 @@
 JSON_LOG_INDENT = int(os.environ.get('JSON_LOG_INDENT')) \
     if os.environ.get('JSON_LOG_INDENT') else None
 
-RESERVED_ATTRS = (
-    'args', 'asctime', 'created', 'exc_info', 'exc_text', 'filename',
-    'funcName', 'levelname', 'levelno', 'lineno', 'module',
-    'msecs', 'message', 'msg', 'name', 'pathname', 'process',
-    'processName', 'relativeCreated', 'stack_info', 'thread', 'threadName')
-
-TOP_LEVEL_ATTRS = ('level', 'service', 'user', 'environment',
-                   'host', 'tags', 'message', 'tenant', 'timestamp',
-                   'path', 'status')
+RESERVED_ATTRS = ('args', 'asctime', 'created', 'exc_info', 'exc_text',
+                  'filename', 'funcName', 'levelname', 'levelno', 'lineno',
+                  'module', 'msecs', 'message', 'msg', 'name', 'pathname',
+                  'process', 'processName', 'relativeCreated', 'stack_info',
+                  'thread', 'threadName')
+
+TOP_LEVEL_ATTRS = ('level', 'service', 'user', 'environment', 'host', 'tags',
+                   'message', 'tenant', 'timestamp', 'path', 'status')
+
+LEVEL_NAMES = {
+    'DEBUG': 'debug',
+    'INFO': 'info',
+    'WARNING': 'warning',
+    'ERROR': 'err',
```

Comment:
Not sure if I like err rather than error and crit rather than critical as it doesn't seem consistent to the others